Activities/Cryptography/UiPath.Cryptography/CryptographyHelper.cs

```diff
@@ -260,5 +315,145 @@ private static int[] GetLegalKeySizes(SymmetricAlgorithm algorithm)
 
             return keySizes.ToArray();
         }
+
+#if NET5_0
+        private static byte[] EncryptAesGcmNet5(byte[] inputBytes, byte[] key, out byte[] result)
+        {
+            byte[] encrypted = new byte[inputBytes.Length];
+            byte[] salt = new byte[PBKDF2_SaltSizeBytes];
+            var tag = new byte[16];
+            var algorithmIV = new byte[12];
+
+            _rng.GetBytes(salt);
+            _rng.GetBytes(algorithmIV);
+
+            using (Rfc2898DeriveBytes pbkdf2 = new Rfc2898DeriveBytes(key, salt, PBKDF2_Iterations))
+            {
+                var Key = pbkdf2.GetBytes(32); //256 bit key
+                var aes = new AesGcm(Key);
+                aes.Encrypt(algorithmIV, inputBytes, encrypted, tag);
+            }
+
+            result = new byte[salt.Length + algorithmIV.Length + encrypted.Length + tag.Length];
+            Buffer.BlockCopy(salt, 0, result, 0, salt.Length);
+            Buffer.BlockCopy(algorithmIV, 0, result, salt.Length, algorithmIV.Length);
+            Buffer.BlockCopy(encrypted, 0, result, salt.Length + algorithmIV.Length, encrypted.Length);
+            Buffer.BlockCopy(tag, 0, result, salt.Length + algorithmIV.Length + encrypted.Length, tag.Length);
+
+            return result;
+        }
+        private static byte[] DecryptAesGcmNet5(byte[] inputBytes, byte[] key, out byte[] decrypted)
```

Comment:
Nitpick: please insert an empty line before this method.

Reply:
added